### DISCUSSION TITLE
Help a newcomer figure out how can they help

### DISCUSSION THREAD

**Discussion by zgeybi** [ClickHouse/ClickHouse#49038] [Q&A] [answered]
Hello everyone!
I am a CS student, and I would like to contribute to ClickHouse project.
I would say I am a junior C++ developer.
I'm just starting out in working on open source projects so I don't really know where to start or what to look for to know where to start, some help in figuring out how can I help would be very appreciated.
If you have a suggestion on something I can start working on, please write me!

**Comment by rschu1ze** [comment, marked as answer]
Hi,

I would say the first step is [to build](https://clickhouse.com/docs/en/development/build) ClickHouse and then to make yourself familiar with the basic functionality.

Afterwards, you could check the list of tasks labeled "[warmup](https://github.com/ClickHouse/ClickHouse/issues?q=is%3Aissue+is%3Aopen+label%3A%22warmup+task%22)" or "[easy](https://github.com/ClickHouse/ClickHouse/issues?q=is%3Aissue+is%3Aopen+label%3A%22easy+task%22)".

**Reply by zgeybi** [reply]
Thank you for your reply!
I will do that
